L/libgifextra/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Giflibextra"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/ashwani-rathee/io-project/raw/main/gifwrapper/gen/libgifextra.zip", "4e1d3e0d1e9df0f30e0dc6c907bb2d574d4c4efec79fb1558451d4c24fd3430b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd libgifextra/
+make
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl")
+]
```

Comment:
Any particular reasons for not building on non-Linux platforms?
```suggestion
platforms = supported_platforms(; exclude=!Sys.islinux)
```